test/testcafe/spec/TerminalView_spec.js

```diff
@@ -70,7 +70,6 @@ const terminalCustomAccessDeniedErrorMessageMock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/introspect')
   .respond(customAccessDeniedErrorMessage);
 
-fixture('Terminal view');
 fixture('Terminal view').meta('v3', true);
```

Comment:
🤔 hopefully this test was still running the whole time

Reply:
I hope so too lol I will find out on this branch, hopefully it doesn't fail 😅